public/app/features/manage-dashboards/state/actions.ts

```diff
@@ -359,10 +359,14 @@ export function getFolderById(id: number): Promise<{ id: number; title: string }
   return getBackendSrv().get(`/api/folders/id/${id}`);
 }
 
-export function deleteDashboard(uid: string, showSuccessAlert: boolean) {
+export function deleteDashboard(uid: string, showSuccessAlert: boolean): Promise<{ id: number; title: string }> {
```

Comment:
Why did you add the promise response?

Reply:
Copy and paste accident; i have deleted it